generator.yml

```diff
@@ -100,16 +100,12 @@ builds:
   - {<< : *arm64,      << : *mainline,         << : *llvm_full,       boot: true,  llvm_version: *llvm_tot}
   # i386: Build disabled (https://github.com/ClangBuiltLinux/linux/issues/1210)
   # - {<< : *i386,       << : *mainline,         << : *llvm_full,       boot: true,  llvm_version: *llvm_tot}
-  # mips and mipsel: Build disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/1)
-  # mips and mipsel: Boot disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/27)
-  # - {<< : *mips,       << : *mainline,         << : *mips_llvm,       boot: false, llvm_version: *llvm_tot}
-  # - {<< : *mipsel,     << : *mainline,         << : *llvm,            boot: false, llvm_version: *llvm_tot}
-  # ppc32, ppc64, and ppc64le: Boot disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/27)
-  - {<< : *ppc32,      << : *mainline,         << : *llvm,            boot: false, llvm_version: *llvm_tot}
-  - {<< : *ppc64,      << : *mainline,         << : *ppc64_llvm,      boot: false, llvm_version: *llvm_tot}
-  - {<< : *ppc64le,    << : *mainline,         << : *llvm,            boot: false, llvm_version: *llvm_tot}
-  # riscv: Boot disabled (https://github.com/ClangBuiltLinux/continuous-integration2/issues/27)
-  - {<< : *riscv,      << : *mainline,         << : *riscv_llvm_full, boot: false, llvm_version: *llvm_tot}
+  - {<< : *mips,       << : *mainline,         << : *mips_llvm,       boot: true,  llvm_version: *llvm_tot}
+  - {<< : *mipsel,     << : *mainline,         << : *llvm,            boot: true,  llvm_version: *llvm_tot}
+  - {<< : *ppc32,      << : *mainline,         << : *llvm,            boot: true,  llvm_version: *llvm_tot}
+  - {<< : *ppc64,      << : *mainline,         << : *ppc64_llvm,      boot: true,  llvm_version: *llvm_tot}
+  - {<< : *ppc64le,    << : *mainline,         << : *llvm,            boot: true,  llvm_version: *llvm_tot}
+  - {<< : *riscv,      << : *mainline,         << : *riscv_llvm_full, boot: true,  llvm_version: *llvm_tot}
```

Comment:
will this depend on the riscv qemu changes?

Reply:
(looks like yes)

Reply:
Yes. See my other comment for why it might be advantageous to just enable booting RISC-V with this pull.